jobbergate-api/jobbergate_api/main.py

```diff
@@ -4,24 +4,25 @@
 import sys
 import typing
 
+import asyncpg
 import sentry_sdk
 from fastapi import FastAPI, Response, status
 from loguru import logger
 from sentry_sdk.integrations.asgi import SentryAsgiMiddleware
 from starlette.middleware.cors import CORSMiddleware
 
-from jobbergate_api import storage
 from jobbergate_api.apps.applications.routers import router as applications_router
 from jobbergate_api.apps.job_scripts.routers import router as job_scripts_router
 from jobbergate_api.apps.job_submissions.routers import router as job_submissions_router
-from jobbergate_api.config import settings
+from jobbergate_api.config import DeployEnvEnum, settings
+from jobbergate_api.storage import database, handle_fk_error
 
 subapp = FastAPI()
 subapp.add_middleware(
     CORSMiddleware, allow_origins=["*"], allow_credentials=True, allow_methods=["*"], allow_headers=["*"],
 )
 
-if settings.SENTRY_DSN:
+if settings.SENTRY_DSN and settings.DEPLOY_ENV != DeployEnvEnum.TEST:
```

Comment:
`DeployEnvEnum.LOCAL` as well, doesn't it?

Reply:
Not necessarily. It's good to test your sentry connection locally.